lib/api/build.rs

```diff
@@ -143,11 +143,17 @@ fn configure_validation(builder: Builder) -> Builder {
             ("SearchPoints.vector_name", "custom = \"crate::grpc::validate::validate_not_empty\""),
             ("SearchBatchPoints.collection_name", "length(min = 1, max = 255)"),
             ("SearchBatchPoints.search_points", ""),
+            // ("SearchPointGroups.search", "required"), // <-- TODO: needs Serialize on the underlying type
```

Comment:
What about those two TODOs?

Reply:
I wanted to ask if validating it manually on the tonic interface would be ok, since it doesn't seem I can just add `required` here. Has anyone faced this problem before?